lib/sendcloud/errors/response_error.rb

```diff
@@ -14,7 +14,7 @@ def initialize(payload:, body:, status:)
       private
 
       def build_message
-        "#{status} #{body[:error][:message]}"
+        "Sendcloud error: #{status} - #{body[:error][:message]}"
```

Comment:
Some of the errors can be a bit cryptic or refer to things like "please contact support", and we store these errors directly on the DeliveryProblem, so I added "Sendcloud" to the message to clarify where the error is actually coming from.